HOWTO-hi.md

```diff
@@ -9,6 +9,6 @@
 
 सवाल पूछने में संकोच न करें; हर योगदानकर्ता ने पहले PR के साथ शुरुआत की। आप हमारे हजारवें हो सकते हैं!
 
-यहां तक ​​कि अगर आप एक अनुभवी ओपन सोर्स योगदानकर्ता हैं, तो ऐसी चीजें हैं जो आपको यात्रा कर सकती हैं। एक बार जब आप अपना पीआर सबमिट कर देते हैं, तो ट्रैविस-सीआई एक लाइनर चलाएगा, अक्सर रिक्ति या वर्णमाला के साथ छोटे मुद्दों को ढूंढता है। यदि आपको एक हरा बटन मिलता है, तो सब कुछ समीक्षा के लिए तैयार है, लेकिन यदि नहीं, तो यह जानने के लिए ट्रैविस लिंक पर क्लिक करें कि लिंटर को क्या पसंद नहीं था। समस्या को ठीक करें और अपने पीआर के लिए एक प्रतिबद्धता जोड़ें।
+यहां तक कि अगर आप एक अनुभवी ओपन सोर्स योगदानकर्ता हैं, तो ऐसी चीजें हैं जो आपको यात्रा कर सकती हैं। एक बार जब आप अपना पीआर सबमिट कर देते हैं, तो गिथब एक्शन एक लिंटर चलाएगा, अक्सर रिक्ति या वर्णमाला के साथ छोटे मुद्दों को ढूंढता है। यदि आपको एक हरा बटन मिलता है, तो सब कुछ समीक्षा के लिए तैयार है, लेकिन यदि नहीं, तो यह जानने के लिए फेल्ड चेक के नीचे "डिटेल्स" पर क्लिक करें कि लिंटर को क्या पसंद नहीं आया। समस्या को ठीक करें और अपने पीआर के लिए एक प्रतिबद्धता जोड़ें।
```

Comment:
Could you give without translation the "GitHub Actions" term? It's a requirement of the issue

> Most instances of the change were simple as we only replaced the noun; therefore nothing should've needed to change. (This is with the assumption that "GitHub Actions" should be treated as a noun and is not localized.)

Reply:
or explain the change?

Reply:
Actually, when I said that I didn't intend for it to be a firm rule. ^-^'
I was just saying what we did, which was with the assumption that nouns don't change.

I believe it's common for languages that have a different character set to modify formal nouns (like "Seth" or "GitHub", not "table"). I'm just using Urdu as an example, but I know with that, nouns are translated by pronunciation to the other character set.

Just verified with a friend who knows Bosnian (can be written in Latin and Cyrillic) and in bs_Cyrl translations, nouns are _often_ translated to use Cyrillic characters.

Reply:
@SethFalco then go ahead?

Reply:
Should I change गिथब एक्शन to "गिथब एक्शन" or to "GitHub Actions"?